saleor/graphql/checkout/mutations.py

```diff
@@ -283,7 +283,11 @@ def save(cls, info, instance: models.Checkout, cleaned_input):
                     raise ValidationError(
                         f"Insufficient product stock: {exc.item}", code=exc.code
                     )
-
+                except ProductNotPublished as exc:
+                    raise ValidationError(
+                        "Can't create checkout with unpublished product.",
```

Comment:
AFAIK Checkout can be already created. This should be something like `Can't add ...`